### PR TITLE
4.x : Add mysql datasource for test setup

### PR DESCRIPTION
**Pull request by deepakrkris**
- [ ]  - add configuration for mysql in CI
- [ ]  - pretest setup for mysql database for CI

### CODE REVIEW

**Comment by rmg** [comment]
@slnode test please

@deepakrkris I've made a temporary change to the CI config so that an appropriate environment is used for running the tests. The change will get overwritten on the next repo scan.

File: package.json [hunk 1]
```diff
@@ -42,6 +42,8 @@
     "grunt-cli": "latest",
     "grunt-cucumber": "^0.3.0",
     "mocha": "^2.4.5",
+    "mysql": "^2.13.0",
```
Comment (rmg) [inline package.json]: Once this is merged the repo scanner will see this dependency and automatically configure the job to run in a MySQL environment.

**Comment by rmg** [comment]
@superkhau @deepakrkris you can ignore the failures on `[cis-jenkins] x64 && linux && nvm,4` and `[cis-jenkins] x64 && linux && nvm,6` - they are from the old configuration and will be replaced by `[cis-jenkins] x64 && linux && nvm && dbs,4` and `[cis-jenkins] x64 && linux && nvm && dbs,6` once this PR is merged.

**Comment by deepakrkris** [comment]
@rmg thanks Ryan

**Pull request merged**